### ISSUE TITLE
[MacOS] App update loop

### ISSUE THREAD

**Issue by Donatas-MN** [mysteriumnetwork/mysterium-vpn-desktop#375]
It looks like something is preventing MacOs app to update for a few times when new update is available. It eventually updates, but it's quite annoying for the user.

See video for details:
https://user-images.githubusercontent.com/91055712/180723701-1fa88af4-9455-4e66-b311-0a43bf01baa3.mov

**Comment by isaackielma** [comment]
[Donatas-MN](https://github.com/Donatas-MN) Are you able to check if this issue persisted on newes app version? https://github.com/mysteriumnetwork/mysterium-vpn-desktop/releases/tag/10.14.2

**Comment by hkup859** [comment]
@isaackielma 
@Donatas-MN 

I can confirm this issue still exists when updating from 10.13.3 to 10.14.2. I noticed in the video you very quickly open the app after quitting and I did that as well when I encountered the issue as well. As a result when testing this again I went very slowly. I noticed 2 things:
1. The VPN automatically minimizes after the update is finished downloading.
2. The VPN automatically re-opened after quitting. I only have the issue shown in the video when I manually restart/re-open it.

I'm not quite sure if the restarting of the app is a feature or a bug, but I can confirm the issue does not happen when I let it restart/re-open itself, all I do is quit the app.

I tested this by installing the software multiple times, ensuring I removed all mysterium related files after an uninstall. I am on MacOS Ventura Version 13.0.1 with an M1 chip.

If anyone else can try and confirm if they encounter the same situation I am getting then I believe we can either close this bug or perhaps rename it to better describe the problem.
